### DISCUSSION TITLE
【吐槽】玩Discord玩的

### DISCUSSION THREAD

**Discussion by AetherLude** [share121/inter-knot#2023] [其他]
![1](https://private-user-images.githubusercontent.com/76843407/357630860-78cdae03-724c-44da-bb5b-fbd23607941a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1OTY0NDQsIm5iZiI6MTcyMzU5NjE0NCwicGF0aCI6Ii83Njg0MzQwNy8zNTc2MzA4NjAtNzhjZGFlMDMtNzI0Yy00NGRhLWJiNWItZmJkMjM2MDc5NDFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDAwNDIyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFiYTg3ZGE3OWQ2OWQ3ZGZhMDA1Y2M2YWViMTg5MDFmNTVjNzg3MjdhMjI2M2JmYTlkZmNhNDJlNjE2NjcyYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1aR2RkbYDL24fGRnjkZmpxyZVAmT9IF-UFhDo-3tyJU)
Discord

**Comment by AetherLude** [comment]
# 别在上午八了

**Comment by VacuolePaoo** [comment]
上完午八了，继续上厕硕

**Comment by Myitian** [comment]
再下去要上早八了，早八上完上午八，再上晚八，接下来没课上了（ /doge
![图片](https://github.com/user-attachments/assets/dfc61446-1824-42f7-a318-a6d34507adb6)
